### ISSUE TITLE
Logout confirmation

### ISSUE THREAD

**Issue by Prince-t** [robertodoering/harpy#423]
There has been a few times where I accidentally click the logout button instead of the about button. Can you make it so that it prompts you to confirm that you want to logout? It is just a tedious process to have to log back in every time it happens.

**Comment by robertodoering** [comment]
Yeah that's a good idea and very simple, thanks for the suggestion :)
